src/DDTrace/Integrations/Laravel/LaravelIntegration.php

```diff
@@ -350,7 +350,7 @@ function ($This, $scope, $args, $loginSuccess) use ($integration) {
 
         // Used by Laravel >= 5.0
         \DDTrace\hook_method(
-            'Illuminate\Auth\Events\Authenticated',
```

Comment:
This is the only core change. Everything else is test code to ensure the fix is working